MIGRATION.md

```diff
@@ -1,3 +1,7 @@
 - Support for Scala 2.7, 2.8 and 2.9 has been dropped. [#86][86]
 
   [86]: https://github.com/sbt/zinc/pull/86
+  
+- sbt.internal.inc.ClassfileManager has been renamed to sbt.internal.inc.ClassFileManager for consistency. [#87][87]
```

Comment:
I would suggest to include these changes in the MIGRATION guide because sbt and pants maintainers will need to migrate it, and it's better that we let them know the changes straight away. We can remove these internal changes for the final MIGRATION guide from 0.13.x to 1.0.x.